package-old.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Maybe it was committed by accident but I don't think we need to keep the -old version of these files around. They'll be available in the commit history if we need to reference them again.